tests/test_templatetags.py

```diff
@@ -2392,3 +2392,277 @@ def test_inner_slot_iteration_nested_with_slot_default_and_outer_scope_variable_
             {'inner': ['ITER2_OBJ1', 'ITER2_OBJ2']} default
             """,
         )
+
+
+class ScopedSlotTest(BaseTestCase):
+    def test_slot_data_as_kwarg(self):
+        @component.register("test")
+        class TestComponent(component.Component):
+            template: types.django_html = """
+                {% load component_tags %}
+                <div>
+                    {% slot "my_slot" data=slot_data %}Default text{% endslot %}
```

Comment:
Oh? I expected the API to be `abc="def" 123="456"`?

Reply:
Hm, maybe you had old version of the MR loaded in the browser. The initial implementation used the `data` object, but then I changed it a couple of days back in https://github.com/EmilStenstrom/django-components/pull/495/commits/3f90d51994ed3143a5c402d680b8bdd71cecfd18. But there was still an old error message, I've fixed that just now!